aikau/src/main/resources/alfresco/renderers/_ItemLinkMixin.js

```diff
@@ -82,10 +86,10 @@ define(["dojo/_base/declare",
          }
          else
          {
-            if (this.currentItem && this.currentItem.node)
+            if (item && item.node)
             {
-               var jsNode = this.currentItem.jsNode;
-               if (jsNode.isLink && this.currentItem.location.site)
+               var jsNode = item.jsNode;
+               if (jsNode.isLink && item.location.site)
```

Comment:
Could item.jsNode be `null` or `undefined` here (i.e. causing `jsNode.isLink` to error)?